app/helpers/common_parameters_helper.rb

```diff
@@ -41,7 +41,7 @@ def use_puppet_default_help link_title = nil, title = _("Use Puppet default")
 
   def hidden_value_field(f, field, value, disabled, options = {})
     if f.object.hidden_value?
-      input = f.password_field(field, :disabled => disabled, :value => value, :class => 'form-control')
+      input = f.password_field(field, :disabled => disabled, :value => value, :class => 'form-control no-stretch')
```

Comment:
This conflicts with the hidden params PR I just merged, I'll fix it when I merge this one.